projection/core/src/test/scala/com/lightbend/lagom/internal/cluster/projections/ProjectionStateSpec.scala

```diff
@@ -36,7 +37,7 @@ class ProjectionStateSpec extends WordSpec with Matchers {
     coordinates002_1.asKey -> coordinates002_1
   )
 
-  val desiredStatus: Map[WorkerKey, Status] = Map(
+  val desiredWorkerStatus: Map[WorkerKey, Status] = Map(
```

Comment:
let's call this `requestedWorkerStatus` to align with the naming we are using on other places